packages/manager/modules/netapp/src/components/select-checkboxes/index.js

```diff
@@ -0,0 +1,17 @@
+import angular from 'angular';
+import 'angular-translate';
+import '@ovh-ux/ng-at-internet';
```

Comment:
**question:** Does AT internet module is needed?

Reply:
Nope (Copy paste again :()